fjord/feedback/static/css/generic_feedback.less

```diff
@@ -98,7 +98,7 @@ header {
       box-sizing: border-box;
       display: inline;
       float: left;
-      margin: 5px 0 0 5px;
+      margin: 0px 0 0 0px;
```

Comment:
You changed the top and left margins here. We don't need to adjust the top part at all, just the left/right parts. So leave the top margin as `5px` and change the left margin to just `0`. i.e. like this:

```
margin: 5px 0 0 0;
```